scripts/webpack.config.js

```diff
@@ -68,7 +68,7 @@ module.exports = {
   devtool: 'source-map',
   externals: {
     react: 'react',
-    'prop-types': 'PropTypes',
```

Comment:
This is the right fix but have you verified if there are any changes in the build?

Reply:
could you elaborate on what you mean by "changes in the build"?

Reply:
as in if looking at what's bundled before/after these changes shows any differences

Reply:
ah! I did! react and prop-types are properly referenced as external dependencies.